build.sbt

```diff
@@ -3822,7 +3822,7 @@ lazy val `engine-runner` = project
       .dependsOn(NativeImage.additionalCp)
       .dependsOn(NativeImage.smallJdk)
       .dependsOn(
-        createEnginePackage
+        createEnginePackageNoIndex
```

Comment:
`engine-runner/buildNativeImage` does not need compiled std libs? Interesting. I didn't know that.

Reply:
[EnsoLibrryFeature](https://github.com/enso-org/enso/blob/a1e01fcc512ecfa489462d7b920a8397c25db8a0/engine/runner/src/main/java/org/enso/runner/EnsoLibraryFeature.java#L19) needs:
- `.enso` sources to search for `polyglot java import`
- JARs to locate the imported classes

It doesn't use `.bindings` - e.g. index.